### ISSUE TITLE
Question about schema.to_json

### ISSUE THREAD

**Issue by lavrd** [quickwit-inc/tantivy#1230]
When I try to convert document to json, fields converted from string into string array.

Example

```rust
#[cfg(test)]
mod tests {
    use tantivy::schema::{Schema, STORED, TEXT};

    #[test]
    fn test() {
        let mut schema_builder = Schema::builder();
        schema_builder.add_text_field("level", TEXT | STORED);
        schema_builder.add_text_field("message", TEXT | STORED);
        let schema = schema_builder.build();

        let doc = schema.parse_document(r#"{"level":"debug","message":"log"}"#).unwrap();
        println!("{}", schema.to_json(&doc));
    }
}
```

What I expected

```json
{"level":"debug","message":"log"}
```

What I have

```json
{"level":["debug"],"message":["log"]}
```

**Comment by fmassot** [comment]
hi @lavrxxx!
Indeed, but this is the expected behavior in tantivy. If you look at `Document` struct, you will see that a document is just a `Vec<FieldValue>`. 
You need to add an extra layer to distinguish the case where you have only one value for a field or several values.

**Comment by lavrd** [comment]
Thanks!

**Comment by fmassot** [comment]
And the fact you raise the question is coming from the fact that documentation is not good right now as pointed it here #875
Closing it now as it will be resolved in #875
